packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -29,7 +29,6 @@ Source0:  foreman.repo
 Source1:  foreman-plugins.repo
 Source2:  foreman.gpg
 Source5:  foreman-client.repo
-Source6:  qpid-copr.repo
```

Comment:
I had the impression that this was still used for the client side. Is it not for katello-agent?

If you're dropping it, you should also remove the file from git.

Reply:
and this is why we have code review :see_no_evil: